meetings/2021-07-21.md

```diff
@@ -0,0 +1,176 @@
+
+Henry: yes, we even have a use case for that [Conditional Access by Payment](https://solid.github.io/authorization-panel/authorization-ucr/#conditional-payment). That is the same thing for say being a university member.
+To get back to our use case: I am happy to just have the resource describing the Group Bob & Alice be private to them. The access control rule can link to the group, but only they know they are part of it. I can make a PR for that.
+
+Matthieu: This sounds like a classic verifiable credential/key-based access where you just need to prove what set you pertain to.
+
+
+### PR Workflow
+
+Pavlik: There is a bit of a problem regarding who works on pull requests and who has to approve them. But it's pretty clear that ACP would be Matthieu, WAC Sarven, WAC+ Henry, but it would be good to have 1 person leading the proposal.
+
+Matthieu: That sounds good, but what do you think would be necessary to make things clearer?
+
+Henry: I think that's more or less what we've been doing.
+
+Pavlik: We could have separate PRs for separate proposals.
+
+Matthieu: I would prefer keeping a PR per use case. I think the current flow does work well to compare the implementation details.
+
+Justin: +1 Pavlik's point or the spirit of it.
+I feel like we needed the specs yesterday and I can't build what I want with the current state of WAC.
+We need to optimize our workflow to get the proposal to a finished state; we need to get there fast.
+The ecosystem needs us to get to a better more robust access control system sooner rather than later.
+
+Henry: I agree that WAC got stuck where it was for 5 to 6 years.
+At least now, the simple Use Cases help us show the limitations of WAC and get feedback, e.g. from Sarven.
+Comparing it to ACP is helping us tweak things a little bit.
+We write out detail by detail the things and figure out problems as we go along.
+We seem to have for example noticed a problem with ACP and WAC both having to copying data. This allows us to figuring out what the problems and disavantages are of each, how to solve them, and perhaps even end up with one system.
+We have not been putting enough time in it, which is why it's very slow. 
+We had the vacations slowing us down right now, and before that we spent 3 weeks discussing [UMA](https://github.com/solid/authorization-panel/blob/main/meetings/2021-06-09.md).
+I suggest we commit this PR and continue with new PRs to continue building on it.
+Matthieu reminded me right before Elf and Justin joined, that [two weeks ago](https://github.com/solid/authorization-panel/commit/e0b77c5976056c42ab7018ade0e766c1fec78d0b) we already agreed to merge and incrementally improve the UC3 PR.
+
+WAC is very easy to implement which is good at the begining. It has been advertised as a good decentralised access control solution, and it is if we tackle a few issues so that it meets expectations.
```

Comment:
```suggestion
WAC is very easy to implement which is good at the beginning. It has been advertised as a good decentralised access control solution, and it is if we tackle a few issues so that it meets expectations.
```